### ISSUE TITLE
[Bug] Error in `scoop alias list` when alias field is absent in config.json

### ISSUE THREAD

**Issue by zatuz** [ScoopInstaller/Scoop#3528]
```
C:\WINDOWS\system32>scoop alias list
Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-IsReadOnly.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\lib\commands.ps1: 22 Carácter: 19
+         $line = ((Get-Content $shim_path) | Where-Object { $_.startsw ...
+                   ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...-IsReadOnly.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-IsReadOnly.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 91 Carácter: 20
+         $content = Get-Content (command_path $_.Name)
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...-IsReadOnly.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 92 Carácter: 9
+         $command = ($content | Select-Object -Skip 1).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 93 Carácter: 9
+         $summary = (summary $content).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-IsFixedSize.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\lib\commands.ps1: 22 Carácter: 19
+         $line = ((Get-Content $shim_path) | Where-Object { $_.startsw ...
+                   ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...IsFixedSize.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-IsFixedSize.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 91 Carácter: 20
+         $content = Get-Content (command_path $_.Name)
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...IsFixedSize.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 92 Carácter: 9
+         $command = ($content | Select-Object -Skip 1).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 93 Carácter: 9
+         $summary = (summary $content).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-IsSynchronized.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\lib\commands.ps1: 22 Carácter: 19
+         $line = ((Get-Content $shim_path) | Where-Object { $_.startsw ...
+                   ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...ynchronized.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-IsSynchronized.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 91 Carácter: 20
+         $content = Get-Content (command_path $_.Name)
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...ynchronized.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 92 Carácter: 9
+         $command = ($content | Select-Object -Skip 1).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 93 Carácter: 9
+         $summary = (summary $content).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-Keys.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\lib\commands.ps1: 22 Carácter: 19
+         $line = ((Get-Content $shim_path) | Where-Object { $_.startsw ...
+                   ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...\scoop-Keys.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-Keys.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 91 Carácter: 20
+         $content = Get-Content (command_path $_.Name)
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...\scoop-Keys.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 92 Carácter: 9
+         $command = ($content | Select-Object -Skip 1).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 93 Carácter: 9
+         $summary = (summary $content).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-Values.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\lib\commands.ps1: 22 Carácter: 19
+         $line = ((Get-Content $shim_path) | Where-Object { $_.startsw ...
+                   ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...coop-Values.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-Values.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 91 Carácter: 20
+         $content = Get-Content (command_path $_.Name)
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...coop-Values.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 92 Carácter: 9
+         $command = ($content | Select-Object -Skip 1).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 93 Carácter: 9
+         $summary = (summary $content).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-SyncRoot.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\lib\commands.ps1: 22 Carácter: 19
+         $line = ((Get-Content $shim_path) | Where-Object { $_.startsw ...
+                   ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...op-SyncRoot.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-SyncRoot.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 91 Carácter: 20
+         $content = Get-Content (command_path $_.Name)
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...op-SyncRoot.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 92 Carácter: 9
+         $command = ($content | Select-Object -Skip 1).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 93 Carácter: 9
+         $summary = (summary $content).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-Count.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\lib\commands.ps1: 22 Carácter: 19
+         $line = ((Get-Content $shim_path) | Where-Object { $_.startsw ...
+                   ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...scoop-Count.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

Get-Content : No se encuentra la ruta de acceso 'C:\Users\zatuz\scoop\shims\scoop-Count.ps1' porque no existe.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 91 Carácter: 20
+         $content = Get-Content (command_path $_.Name)
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\zatuz\...scoop-Count.ps1:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 92 Carácter: 9
+         $command = ($content | Select-Object -Skip 1).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

No se puede llamar a un método en una expresión con valor NULL.
En C:\Users\zatuz\scoop\apps\scoop\current\libexec\scoop-alias.ps1: 93 Carácter: 9
+         $summary = (summary $content).Trim()
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull
```

Count
IsFixedSize
IsReadOnly
IsSynchronized
Keys
SyncRoot
Values

**Comment by marcosbozzani** [comment]
The same thing was happening to me.  The workaround was to add an empty `alias` config like this: 

` scoop config alias @{}`

I don't why the original `config.json` was missing this key.

Hope it helps!

**Comment by Neurognostic** [comment]
> The same thing was happening to me. The workaround was to add an empty `alias` config like this:
> 
> ` scoop config alias @{}`
> 
> I don't why the original `config.json` was missing this key.
> 
> Hope it helps!

Thank you for this!

I have to do this workaround every time scoop is installed or the config is wiped. This was especially apparent in the latest release of [scoop-completion](https://github.com/Moeologist/scoop-completion/) which tries to get the aliases during [tab] completion.

See https://github.com/Moeologist/scoop-completion/issues/13

**Comment by davidhaymond** [comment]
I ran into this problem today, but there doesn't seem to be any movement towards fixing this here. I'll go ahead and take a look at it myself and file a PR.

**Comment by BinToss** [comment]
I ran into this issue while writing up a Pull Request for scoop-completion to use Register-ArgumentCompleter because I had though it would output a list of shims as well as aliases. It only outputs the latter.

> The same thing was happening to me. The workaround was to add an empty `alias` config like this:
> 
> ` scoop config alias @{}`
> 
> I don't why the original `config.json` was missing this key.
> 
> Hope it helps!

This workaround is still necessary for `scoop alias list` to work.

**Comment by rashil2000** [comment]
Closed in #4737
